ansible/roles/ssc/templates/ssc-deploy.yml.j2

```diff
@@ -0,0 +1,135 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: ssc-data
+  namespace: sagasrv-ssc
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: {{ ssc.volume_size }}
+  storageClassName: saga-default
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ssc-init-script
+  namespace: sagasrv-ssc
+data:
+  init-validator.sh: |
+    #!/bin/sh
+    if [ ! -f /root/.ssc/config/priv_validator_key.json ]; then
+      echo "Initializing validator..."
+      sscd init ssc-validator --home /root/.ssc --chain-id {{ ssc.chain_id }} --default-denom {{ ssc.denom }}
+      
+      echo "Adding validator account..."
+      echo "$VALIDATOR_MNEMONIC" | sscd keys add ssc-validator --home /root/.ssc --keyring-backend test --recover --no-backup
+      
+      echo "Adding genesis account..."
+      sscd add-genesis-account ssc-validator 1000000000000{{ ssc.denom }} --home /root/.ssc --keyring-backend test
+      
+      echo "Creating gentx..."
+      sscd genesis gentx ssc-validator 1000000000{{ ssc.denom }} --home /root/.ssc --keyring-backend test --chain-id {{ ssc.chain_id }}
+      
+      echo "Collecting gentxs..."
+      sscd genesis collect-gentxs --home /root/.ssc
+      
+      echo "Validator initialization complete"
+    else
+      echo "Validator already initialized"
```

Comment:
So, when not initialized we should do a few things
- Create validator key from mnemonic: `echo "$VALIDATOR_MNEMONIC" | sagaosd init $MONIKER --chain-id $CHAINID --recover`. Now if we want to support fullnodes too, you can init without the mnemonic.
- If `ssc_genesis_url` is set download it
- If not set you can generate it assuming it's a single validator

Reply:
Your script works, but only on devnet (1 validator). We also want to support:
1. Other networks (mainnet, testnet, staging etc)
2. Fullnodes